src/components/Projects/Projects.jsx

```diff
@@ -121,6 +123,44 @@ const Projects = function(props) {
     await props.postNewProject(name, category);
   };
 
+  // Fetch autocomplete suggestions
+  useEffect(() => {
+    const fetchSuggestions = async () => {
```

Comment:
The fetchSuggestions function is being redefined on every render. While this is usually not a performance issue, you can optimize it by memoizing the function with useCallback if necessary, especially if the component grows in complexity or has performance bottlenecks.
const fetchSuggestions = useCallback(async () => { // Same logic }, [debouncedSearchName, props.getUserByAutocomplete]);

Reply:
Fixed it @navya9989. Could you please review the PR again?
Thanks